tuna/machine.py

```diff
@@ -186,22 +207,22 @@ def get_gpu(self, idx):
 
     return self.gpus[idx]
 
-  def get_cpus(self):
-    """return cpu details array"""
-    if not self.cpus:
-      self.get_properties()
-
-    return self.cpus
-
-  def parse_agents(self):  #pylint: disable=too-many-locals
+  def parse_agents(self) -> dict:  #pylint: disable=too-many-locals
```

Comment:
shouldn't disable pylint

Reply:
this should be addressed in a different PR

Reply:
I havnot edited nor changed this condition it is retained as it is.